File: Sem4/MPP_Templates/Zboruri/FlightNetworking/src/main/java/dto/ClientDTO.java
package dto;

import java.io.Serializable;

public class ClientDTO implements Serializable {
    private String nume;
    private String adresa;

    public ClientDTO(String nume, String adresa) {
        this.nume = nume;
        this.adresa = adresa;
    }

    public String getNume() {
        return nume;
    }

    public String getAdresa() {
        return adresa;
    }
}

File: Sem4/MPP_Templates/ZboruriCSharp/networking/ClientDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace networking
{
    [Serializable]
    public class ClientDTO
    {
        public string nume { get; set; }
        public string adresa { get; set; }

        public ClientDTO(string nume, string adresa)
        {
            this.nume = nume;
            this.adresa = adresa;
        }
    }
}
